.eslintrc.js

```diff
@@ -1,81 +1,64 @@
 module.exports = {
-    'env': {
-        'browser': true,
-        'es6': true,
-        "jest/globals": true,
-        'jquery': true,
-        "node": true
+    env: {
+        browser: true,
+        es6: true,
+        'jest/globals': true,
+        jquery: true,
+        node: true
     },
-    'parser': "babel-eslint",
-    'parserOptions': {
-        "ecmaVersion": 6,
+    parser: 'babel-eslint',
+    parserOptions: {
+        ecmaVersion: 6
     },
-    "plugins": ["jest", "flowtype", "import", "jsx-a11y"],
-    'extends': [
+    plugins: ['jest', 'flowtype', 'import', 'jsx-a11y', 'react-hooks'],
+    extends: [
         'eslint:recommended',
-        "plugin:import/errors",
-        "plugin:flowtype/recommended",
-        "plugin:jsx-a11y/recommended",
-        "prettier"
+        'plugin:import/errors',
+        'plugin:flowtype/recommended',
+        'plugin:jsx-a11y/recommended',
+        'prettier'
     ],
-    'rules': {
+    rules: {
         'no-global-assign': 'error',
-        'indent': [
-            'error',
-            4,
-            {SwitchCase: 1}
-        ],
-        'linebreak-style': [
+        indent: ['error', 4, { SwitchCase: 1 }],
+        'linebreak-style': ['error', 'unix'],
+        'no-cond-assign': 'error',
+        'no-console': [
             'error',
-            'unix'
+            {
+                allow: ['error', 'info']
+            }
         ],
-        'no-cond-assign': 'error',
-        'no-console': ['error', {
-            'allow': ['error', 'info']
-        }],
         'no-empty': 'error',
         'no-extra-semi': 'error',
         'no-fallthrough': 'error',
         'no-redeclare': 'error',
         'no-undef': 'error',
         'no-unused-vars': 'error',
-        'quotes': [
-            'error',
-            'single'
-        ],
-        'semi': [
-            'error',
-            'always'
-        ],
-        'curly': [
-            'error',
-            'all'
-        ],
-        'camelcase': [
+        quotes: ['error', 'single'],
+        semi: ['error', 'always'],
+        curly: ['error', 'all'],
+        camelcase: [
             'error',
             {
-                'properties': 'always'
+                properties: 'always'
             }
         ],
-        'eqeqeq': [
-            'error',
-            'smart'
-        ],
-        'one-var-declaration-per-line': [
-            'error',
-            'always'
-        ],
-        'new-cap': 'error'
+        eqeqeq: ['error', 'smart'],
+        'one-var-declaration-per-line': ['error', 'always'],
+        'new-cap': 'error',
+        'react-hooks/rules-of-hooks': 'error',
+        'react-hooks/exhaustive-deps': 'warn'
```

Comment:
Prettier formatted this file on save. The only changes other than that, are the two lines above and line 13.

Reply:
Haha! It happens. By the way, I suspect it was your editor that ran prettier on this file because they way we have set up `prettier` in `package.json` is on a strict path directory. 
I think this is fuel for the argument that we should use `.prettierignore` or something global instead.